modules/manage/pages/schema-reg/schema-reg-overview.adoc

```diff
@@ -21,6 +21,8 @@ The Schema Registry is built directly into the Redpanda binary. It runs out of t
 
 **Serialization format**: A serialization format defines how data is converted into bytes that are transmitted and stored. Serialization, by producers, converts an event into bytes. Redpanda then stores these bytes in topics. Deserialization, by consumers, converts the bytes of arrays back into the desired data format. Redpanda's Schema Registry supports Avro, Protobuf, and JSON serialization formats.
 
+**Normalization**: Normalization is the process of converting a schema into its simplest form. When a schema is normalized, it can be compared and considered equivalent to another schema even if they contain minor syntactic differences. Schema normalization allows you to more easily manage schema versions and compatibility by prioritizing meaningful logical changes. Normalization during both schema registration and lookup for a subject is supported for Avro and Protobuf.
```

Comment:
```suggestion
**Normalization**: Normalization is the process of converting a schema into a canonical form. When a schema is normalized, it can be compared and considered equivalent to another schema that may contain minor syntactic differences. Schema normalization allows you to more easily manage schema versions and compatibility by prioritizing meaningful semantic changes. Normalization is supported during both schema registration and lookup for a subject is supported for Avro, Json, and Protobuf.
```